Airports/EGDR/Positions.txt

```diff
@@ -1,3 +1,3 @@
-EGDR_APP:Culdrose Approach:134.050:DRA:A:EGDR:APP:-:-:7030:7045:N050.05.09.000:W005.15.17.000::
+EGDR_APP:Culdrose Approach:134.050:DRR:A:EGDR:APP:-:-:7030:7045:N050.05.09.000:W005.15.17.000::
```

Comment:
Culdrose Radar when providing radar service?

Reply:
No Military units always use Approach as the callsign.